backends/advanced/src/scripts/cleanup_state.py

```
@@ -0,0 +1,912 @@
+            }
+            messages_data.append(message_dict)
+
+        stats.chat_messages_count = len(messages_data)
+
+        output_path = self.backup_path / "chat_messages.json"
+        with open(output_path, 'w') as f:
+            json.dump(messages_data, f, indent=2, default=str)
+
+        logger.info(f"Exported {stats.chat_messages_count} chat messages")
+
+    async def _export_audio_wav(self, stats: CleanupStats):
```

Ruff (0.14.11)

Unused method argument: `stats`

(ARG002)